config/environments/test.rb

```diff
@@ -0,0 +1,60 @@
+require "active_support/core_ext/integer/time"
```

Comment:
- [ ] Please make sure there are proper tests present for the important methods as per the requirements.

Reply:
This issue is well resolved! ✔️